Examples/Physics_applications/spacecraft_charging/inputs_test_rz_spacecraft_charging_picmi.py

```diff
@@ -79,10 +80,15 @@ def save_normalized_vacuum_Efields(
         assert np.all(abs(rho[...]) < 1.0e-11)
 
         # Record fields
-        Er = ExWrapper(include_ghosts=True)[:, :]
-        self.normalized_Er = Er[...] / q_v
-        Ez = EzWrapper(include_ghosts=True)[:, :]
-        self.normalized_Ez = Ez[...] / q_v
+        # TODO: add guard cells
```

Comment:
The MultiFab always includes its guard cells by default:
https://warpx.readthedocs.io/en/latest/usage/workflows/python_extend.html#fields